src/ast.rs

```diff
@@ -7297,6 +7310,29 @@ impl Parse for SpecialName {
                     tail,
                 ))
             }
+            b"GT" => {
+                match tail.peek() {
+                    Some(b'n') => {
+                        let tail = consume(b"n", tail).unwrap();
+                        let (base, tail) = Encoding::parse(ctx, subs, tail)?;
+                        Ok((
+                            SpecialName::NonTransactionClone(Box::new(base)),
+                            tail,
+                        ))
+                    }
+                    // Different letters stand for different types of transactional cloning,
+                    // so they should be treated the same.
+                    Some(b't')
+                    | _ => {
```

Comment:
Accepting `None` here is wrong.

Reply:
Overlooked that, thanks for letting me know